src/jobs/index.html

```diff
@@ -61,201 +61,55 @@
-    <li>Mobile applications....soon™</li>
-  </ul>
-  <p>Our codebase was recently completely rewritten to use React/Redux/Typescript which means you won’t be maintaining legacy code here. 😃 Everything we do is open-source and licensed under MIT. Check it out: <a target="_blank" rel="noopener" href="https://github.com/mycryptohq/">https://github.com/mycryptohq/</a>.</p>
-  <h5>Things You Probably Care About</h5>
-  <ul>
+    <li>Desktop Application (Electron): <a target="_blank" rel="noopener" href="https://download.mycrypto.com/">https://download.mycrypto.com/</a></li>
+    <li>Ambo (iOS now, Android soon): <a target="_blank" rel="noopener" href="https://ambo.io/">ambo.io</a></li>
+    <li>Developer/Ecosystem Tools: <a target="_blank" rel="noopener" href="https://overflow.mycrypto.com/">overflow.mycrypto.com</a></li>
+    <li>CryptoScamDB: <a target="_blank" rel="noopener" href="https://cryptoscamdb.org/">cryptoscamdb.org</a></li>
+    <li>FindETH: <a target="_blank" rel="noopener" href="https://findeth.io/">https://findeth.io/</a></li>
+    </ul>
+    <p>Our code is fresh and we primarily use React & Typescript, meaning you won’t be maintaining legacy code. 😃 Everything we do is open-source and licensed under MIT. Check it out: <a target="_blank" rel="noopener" href="https://github.com/mycryptohq/">https://github.com/mycryptohq/</a>.</p>
+    
+    <h5>Things You Probably Care About</h5>
+    <ul>
     <li>We’re a decentralized (remote) team 🚀 with core operations based in Los Angeles.</li>
-    <li>This is a fast-paced, sink-or-swim environment.</li>
+    <li>We have team members in the US, Canada, UK, Netherlands, France, and Slovenia.</li>
+    <li>Weekly check-in calls are between 9am-11am Pacific Time. Everything else is async.</li>
+    <li>We move fast and encourage autonomy. This means you are responsible for managing your time and priorities. It also means you get to work on things you want to work on, when you want to work on them.</li>
     <li>All positions begin as a contract role, part-time or full-time.</li>
-    <li>Most positions will move to full-time employment (if you are able to be employed in the US).</li>
-    <li>Unique perks are available for full-time folks, like traveling to badass conferences around the world & <a target="_blank" rel="noopener" href="https://foreverlabs.com/">banking your stem cells</a>. </li>
-    <li>Each pay-period you can choose to be paid in ETH or USD. 🤑</li>
-  </ul>
-  <h5>Things We Care About</h5>
-  <ul>
+    <li>Most positions will move to full-time employment (if you are able to be employed in the US)</li>
+    <li>Unique perks are available for those who are part of our team full-time, regardless of contractor or employee status. We send you to the amazing crypto-conferences around the world &amp, <a target="_blank" rel="noopener" href="https://foreverlabs.com/">bank your stem cells</a>, and offer equity (even for those located outside the US.)</li>
```

Comment:
Unique perks are available for those who are part of our team full-time, regardless of contractor or employee status. Sometimes we'll send you to events around the world, we'll help you bank your stem cells, and may offer equity (even for those located outside the US).